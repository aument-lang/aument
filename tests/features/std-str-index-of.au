print str::index_of("abcdef", "def");
print str::index_of("", "a");