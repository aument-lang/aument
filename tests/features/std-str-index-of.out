int;3
int;-1